app/codes/transactionmanager.py

```diff
@@ -325,6 +327,15 @@ def econvalidator(self):
                 self.transaction['specific_data']['asset1_number'], token1mp)
             sender1valid = False
             sender2valid = False
+            sc_created_token1 = is_token_sc_created(tokencode1)
+            if(sc_created_token1):
```

Comment:
@Vaibhav083 let's not use parentheses for if statement in python
[https://peps.python.org/pep-0008/](https://peps.python.org/pep-0008/)